integrations/vello_svg/src/lib.rs

```diff
@@ -175,8 +174,47 @@ fn paint_to_brush(paint: &usvg::Paint, opacity: usvg::Opacity) -> Option<Brush>
             color.blue,
             opacity.to_u8(),
         ))),
-        usvg::Paint::LinearGradient(_) => None,
-        usvg::Paint::RadialGradient(_) => None,
+        usvg::Paint::LinearGradient(gr) => {
+            let stops: Vec<Color> = gr
+                .stops
+                .iter()
+                .map(|stop| {
+                    vello::peniko::Color::rgba8(
+                        stop.color.red,
+                        stop.color.green,
+                        stop.color.blue,
+                        stop.opacity.to_u8(),
+                    )
+                })
+                .collect();
+            let start: vello::kurbo::Point = gr.transform.apply(gr.x1, gr.y1).into();
+            let end: vello::kurbo::Point = gr.transform.apply(gr.x2, gr.y2).into();
+            let gradient =
+                vello::peniko::Gradient::new_linear(start, end).with_stops(stops.as_slice());
+            Some(Brush::Gradient(gradient))
+        }
+        usvg::Paint::RadialGradient(gr) => {
+            let stops: Vec<Color> = gr
+                .stops
+                .iter()
+                .map(|stop| {
+                    vello::peniko::Color::rgba8(
+                        stop.color.red,
+                        stop.color.green,
+                        stop.color.blue,
+                        stop.opacity.to_u8(),
+                    )
+                })
+                .collect();
+            let center: vello::kurbo::Point = gr.transform.apply(gr.cx, gr.cy).into();
+            // TODO: SVG has a scaleX and scaleY - But peniko::Gradient::new_radial only takes a radius.
```

Comment:
Special attention to this - Maybe worth filing an issue for Vello to introduce radiusX and radiusY for radial gradients? SVG has rx and ry.

Reply:
Based on the SVG spec (https://developer.mozilla.org/en-US/docs/Web/SVG/Element/radialGradient#attributes), there are actually two centers, start and end, which are represented by (fx, fy) and (cx, cy) respectively. In peniko/vello, this type of gradient can be created with the `Gradient::new_two_point_radial` constructor which does take two radii. Unfortunately, it doesn't look like usvg exposes the `fr` attribute (the focal point, or start circle radius) so I'd just pass the `r` attribute for both.

This would look something like `Gradient::new_two_point_radial((gr.fx, gr.fy), gr.r, (gr.cx, gr.cy), gr.r)`.

For the transform, we can't apply it directly to the gradient properties because non-uniform scales or skews will change the geometry of the gradient. The vello rendering functions take an optional `brush_transform` parameter to handle this. We probably want to change the `paint_to_brush` function to return `Option<(Brush, Option<Affine>)>` so that gradient transforms can be passed along.